lib/dns.js

```diff
@@ -301,7 +301,13 @@ function resolve(hostname, rrtype, callback) {
 
 
 function getServers() {
-  return cares.getServers();
+  const ret = cares.getServers();
+  return ret.map((val) => {
+    if (!val[1] || val[1] === 53) return val[0];
```

Comment:
I just noticed this, IMHO we should format IPv6 with `[]`, so need to move the `const host` up:
e.g.
```js
return ret.map(([host, port]) => {
  if (isIP(host) === 6) host = `[${host}]`;
  return (!port || port === 53) host : `${host}:${port}`;
}
```
But that will make this `semver-major` (since till now IPv6 would be represented without `[]`)